resources/creatingchannels.md

```diff
@@ -0,0 +1,58 @@
+# Creating Channels with CompSciBot
+
+- [Creating Semester Channels](#creating-semester-channels)
+- [Removing Semester Channels](#removing-semester-channels)
+
+---
+
+## Creating Semester Channels
+
+To create the semester channels, first you need to get the csv of the classes for the semester.
+
+There are many ways to do this, but keep in mind the csv is parse by order, not by column name (for example, the fourth column is always the class number).
+
+### Getting the csv
+
+1. **Log into** the [EMU registration portal](https://bannerweb.emich.edu/pls/banner/twbkwbis.P_GenMenu?name=bmenu.P_RegMnu) and go into Student Services > Registration > Look-up Classes > [Select Semester] > Advanced Search
```

Comment:
Makes sense, will switch it for new documentation.

Reply:
In the new documentation, it isn't really necessary because the Chrome extension automatically opens the correct URL

https://github.com/EMU-Compsci-Discord/compsci-class-scraper/blob/2658513389b40d14c436a519c4bcfed1ac22a537/src/popup.ts#L6

https://github.com/EMU-Compsci-Discord/compsci-class-scraper/blob/2658513389b40d14c436a519c4bcfed1ac22a537/src/popup.ts#L48